pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "pandas>=1.1.5",
     "scikit-learn>=1.0",
     "importlib-metadata >= 1.0; python_version < '3.8'",
+    "importlib-resources; python_version < '3.9'",
```

Comment:
[importlib.resources.files](https://docs.python.org/3.11/library/importlib.resources.html#importlib.resources.files) is new in 3.9, [`importlib-resources`](https://pypi.org/project/importlib-resources/) is the backport